warehouse/query-core/src/main/java/datawave/query/language/processor/lucene/CustomAnalyzerQueryNodeProcessor.java

```diff
@@ -252,6 +359,11 @@ protected QueryNode tokenizeNode(QueryNode node, String text, String field) thro
                     }
                 }
                 
```

Comment:
put a comment here as to the state at this point in the code: namely n's text is different than ours and hence we will create an or node with the two children....